src/logging/errorCodes.ts

```diff
@@ -2,4 +2,22 @@ export type ErrorCodes = (typeof ErrorCodes)[keyof typeof ErrorCodes];
 
 export const ErrorCodes = {
     UNHANDLED_EXCEPTION: 'UNHANDLED_EXCEPTION',
+    EMAIL_MISMATCH: 'EMAIL_MISMATCH',
```

Comment:
Talked this over with @mattbodle and @rmi22186 and I think we may need to review the existing error codes for the WSDK and see how they overlap. There's a lot of error codes here that are specific to the mP SDK modules, but may not be relevant to our metrics and logs in the dashboard.